README.md

```diff
@@ -91,6 +91,11 @@ optional arguments:
                         This argument is expected in JSON notation. For a list
                         of possible options, refer to the ffmpeg-imageio docs.
                         Default: '{"codec": "libx264"}'.
+  --track-config        SORT arguments for bounding-box detection tracking, or 
+                        None if tracking should not be used.
+                        This argument is expected in JSON notation. For a list
+                        of possible options, refer to the original SORT repository.
```

Comment:
SORT is quite hard to discover through Google, so I'd rather link directly to its options or describe them, maybe based on the help texts provided here: https://github.com/abewley/sort/blob/master/sort.py#L261-L267